1228-public_page/anonymous_user_views_landing_page_for_registration.feature

```diff
@@ -1,32 +1,32 @@
-Feature: Anonymous User View Public Page for Publication
-
-    @881
-    Scenario Outline: Anonymous User View Public Page for Publication
-        Given the Anonymous User has opened NVA
-        And they see the "<Page>"
-        When they click to view a Publication
-        Then the Public Page for Publication is opened
-        And they see page fields for
-            | Title                  |
-            | Abstract               |
-            | NPI                    |
-            | Keywords               |
-            | Publication date       |
-            | Primary language       |
-            | Projects               |
-            | Publication type       |
-            | Publication subtype    |
-            | Values per sybtype     |
-            | Clickable Contributors |
-            | Clickable emails       |
-            | Affiliations           |
-            | File download link     |
-            | ORCID                  |
-            | DOI link               |
-            | Licence                |
-            | "<Licence link>"       |
-
-        Examples:
-            | Page        | Licence link                                                            |
-            | Start Page  | https://creativecommons.org/licenses/by/4.0/deed.no                     |
-            | Public Page | https://dlr.unit.no/licenses/ntnu-alle-rettigheter-forbeholdt-forfatter |
+Feature: Anonymous User views Landing Page for Registration
+
```

Comment:
Nå syns jeg kanskje det ble litt rart med egen fil for visning av Filer. Enige i at alle de scenarioene kunne blitt flyttet hit? @sarahse @Eiriknil

Reply:
jeg synes egentlig det er fint med en egen fil for filer pga det er så mange scenario. Det skal jo være scenario for annen funksjonalitet på Landing Page også, og jeg tror det fort blir for mye hvis alle skal inn her

Reply:
Enig at vi kan ha egen feature for filer